<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addinfo</class>
 <widget class="QWidget" name="addinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#addinfo{
	background-color: rgb(33, 36, 43);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>22</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox#groupBox{background-color: qradialgradient(cx:0, cy:0, radius:1, fx:0.1, fy:0.1, stop:0 rgb(75, 6, 89),  stop:1 rgb(45, 14, 48));
border:none;}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>企业信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>学生信息</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>24</number>
       </property>
       <item>
        <widget class="QLabel" name="company_or_student_name_label">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string>企业名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="company_or_student_name">
         <property name="maximumSize">
          <size>
           <width>249</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>31</number>
       </property>
       <item>
        <widget class="QLabel" name="admitreq_or_exam_label">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string>学历要求</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="admitreq_or_exam">
         <property name="maximumSize">
          <size>
           <width>292</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="hirenum_or_stuid_label">
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string>招聘人数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="hirenum_or_stuid">
         <property name="maximumSize">
          <size>
           <width>249</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>35</number>
       </property>
       <item>
        <widget class="QLabel" name="publisher_or_ifjob_label">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string>发布人</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="publisher_or_ifjob">
         <property name="maximumSize">
          <size>
           <width>292</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>238</number>
     </property>
     <item>
      <widget class="QLabel" name="companytype_or_major_label">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>企业类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="companytype_or_major">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>150</number>
     </property>
     <property name="rightMargin">
      <number>150</number>
     </property>
     <item>
      <widget class="QPushButton" name="back_btn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(134, 117, 255);
}
QPushButton:hover{
background-color: rgba(114, 129, 214, 59);
}
QPushButton:pressed{
background-color: ;
	color: rgba(21, 224, 255,59);
}</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="confirm_btn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(134, 117, 255);
}
QPushButton:hover{
background-color: rgba(114, 129, 214, 59);
}
QPushButton:pressed{
background-color: ;
	color: rgba(21, 224, 255,59);
}</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
